### ISSUE TITLE
Privacy & Co

### ISSUE THREAD

**Issue by sheepyhollow** [ClockGU/documentation#9]
Wie ist denn der Status (Planung oder Implementation) der Datenschutz-Erfordernisse?

* Einsicht der Daten (eigentlich kann ich immer alles sehen)
* Löschen aller Daten (Account resetten)
* Löschen spezifischer Daten (darf ich z.B. exportierte Reports/Schichten löschen?)
* Export aller Daten (Auskunftsrecht)

**Comment by sheepyhollow** [comment]
Sind die Daten in der DB eigentlich verschlüsselt? Oder könntet ihr direkt die Einträge lesen?

**Comment by chgad** [comment]
Du meinst ob ein Administrator die Daten eines Nutzers einsehen kann? Aka: Ich sehe das der Nutzer Bertram Bühner, Schicht A, Schicht B und Schicht C in Vertrag "Büroarbeit" gestochen hat?

Ja, das ist möglich. Ist das ein Problem?

**Comment by chgad** [comment]
> 
    Einsicht der Daten (eigentlich kann ich immer alles sehen)
    Löschen aller Daten (Account resetten)
    Löschen spezifischer Daten (darf ich z.B. exportierte Reports/Schichten löschen?)
    Export aller Daten (Auskunftsrecht)

1. Ja.
2. Jain. Der Account wird inaktiv. Das ist bis jetzt die Absprache mit Herrn Scharf
3. Bis zum Export ist das Löschen jeder nicht exportierten Schicht gestattet - anschließend nicht mehr. Wo stellt das ein Problem da?
4. Noch nicht implementiert, aber im Blick siehe [hier](https://github.com/ClockGU/clock-backend/issues/58)

**Comment by sheepyhollow** [comment]
> Ja, das ist möglich. Ist das ein Problem?

Nein - aber ich würde das dann in der Doku entsprechend transparent machen. Ist aktuell auch so umgesetzt. Aber ich würde dann nicht behaupten, dass es verschlüsselt wäre oder so. Im Moment ist das etwas offen gelassen.

**Comment by sheepyhollow** [comment]
https://clockgu.gitbook.io/technical/benutzerhandbuch/privacy#wer-kann-meine-daten-einsehen

**Comment by sheepyhollow** [comment]
> 3. Bis zum Export ist das Löschen jeder nicht exportierten Schicht gestattet - anschließend nicht mehr. 

Die DSGVO macht afaik keine Aussagen zur Löschung spezifischer Daten. Insofern passt das so, wie es ist.

**Comment by chgad** [comment]
> Nein - aber ich würde das dann in der Doku entsprechend transparent machen. Ist aktuell auch so umgesetzt. Aber ich würde dann nicht behaupten, dass es verschlüsselt wäre oder so. Im Moment ist das etwas offen gelassen.

Wessen Datenbank ist denn überhaupt versclüsselt? Irgendwie muss doch der Support auch auf die daten des anfordernden Nutzers zugriff haben können. @mimischi; versteh ich hier was falsch?

**Comment by mimischi** [comment]
Ja, also verschlüsselt macht keinen Sinn, wenn wir immer mal wieder rein sehen müssen falls etwas nicht funktioniert. Und so lange wir nichts mit den Daten machen, sollte es DSGVO-konform bleiben.

**Comment by sheepyhollow** [comment]
ja, rechtlich ist das kein Problem. Geht mir nur drum, was wir den Nutzern erzählen. Aber dann formuliere ich das entsprechend und nehme den Verschlüsselungs-Teil raus. Danke!
